mkdocs-project-dir/docs/Account_Services.md

```diff
@@ -1,68 +1,137 @@
+to access core services like Portico and HR systems.
 
-You will need a UCL user name and password. These are the same credentials used to access core services like Portico and HR systems. 
+<!--- TODO: (them): perhaps links for Portico and "HR systems"? --->
 
 ### Application process
 
 The application process has these steps:
 
 1. Enter your UCL username and password to access the application form.
-2. Complete the application form, reading the instructions carefully. Tip: Hover your mouse over text boxes for more information.
-3. When you successfully submit the form, your sponsor will be sent an email asking them to approve your application. If you do not require a sponsor, your application will be automatically approved.
+
+2. Complete the application form, reading the instructions carefully. Tip: hover
+   your mouse over text boxes for more information.
+
+3. When you successfully submit the form, your sponsor will be sent an email message
+   asking them to approve your application. If you do not require a sponsor,
+   your application will be automatically approved.
+
 4. Your sponsor should click on the link in the email, log in and approve the account.
-5. You will then receive an email when your account is approved if you have a sponsor. (This does not mean your account has been created yet).
-6. You should receive an email once we have created your account. Please note that there may be a delay of up to one working day between an application being approved and your account being created.
 
-If your sponsor does not receive the email to approve your account, send them the link to your application directly (it will look like dashboard.rc.ucl.ac.uk/computing/requests/xxxx) and they will have a button at the top to approve it.
+5. You will then receive an email when your account is approved, if you have a
+   sponsor. (This does not mean your account has been created yet).
+
+6. You should receive an email once we have created your account. Please note
+   that there may be a delay of up to one working day between an application
+   being approved and your account being created.
+
+If your sponsor does not receive the email to approve your account, send them
+the link to your application directly (it will look like
+`dashboard.rc.ucl.ac.uk/computing/requests/xxxx`) and there will be a button at
+the top to approve it.
+
+Please note: most delays in the application process are caused by sponsors
+missing or not receiving the approval email, so check with your sponsor first if
+your application is stuck waiting for approval. Research Computing can only
+create your account after it is approved.
 
-Please note: most delays in the application process are caused by sponsors missing or not receiving the approval email, so check with your sponsor first if your application is stuck waiting for approval. Research Computing can only create your account after it is approved. 
+## Accounts for Visitors
```

Comment:
Don't think that needs capitalising: distinction between generic visitor and UCL Visitor status.